løsningsforslag/R1/R1_18V/R1_18V_lf.tex

```diff
@@ -0,0 +1,408 @@
+% Formålet med å legge header til et eget dokument er å garantere at
+% oppsettet av dokumentene er likt for alle løsningsforslagene.
+% I headeren skjer følgende:
+% (1) Dokumentet blir startet
+% (2) Pakker blir importert
+% ---------------------------------------------------------------------
+\input{../../header.tex}
+
+
+% ---------------------------------------------------------------------
+% DOKUMENTVARIABLER
+% ---------------------------------------------------------------------
+\newcommand{\fagkode}{R1}
+\newcommand{\semesteraar}{våren 2018}
+\newcommand{\forfatter}{Anita G.}
+\newcommand{\dokumenttittel}{Løsningsforslag -- Eksamen \fagkode, \semesteraar}
+
+\usepackage{siunitx}
+
+
+% Set til 'true' og oppgi logo dersom du vil bruke en logo
+\newboolean{bruklogo}
+\setboolean{bruklogo}{false}
+\newcommand{\logonavn}{}
+
+
+
+% ---------------------------------------------------------------------
+% SETUP
+% Formålet med å legge setup til et eget dokument å garantere at headers,
+% footers, og øverste del av dokumentet er likt for alle
+% løsningsforslagene.
+% ---------------------------------------------------------------------
+\input{../../setup.tex}
+
+
+% ---------------------------------------------------------------------
+% DOKUMENTSTART - Skriv løsningsforslaget nedenfor
+% ---------------------------------------------------------------------	
+\section*{Del 1 - uten hjelpemidler}
+\subsection*{Oppgave 1}
+\begin{easylist}[enumerate]
+	\ListProperties(Style2*=,Numbers=a,Numbers1=l,FinalMark={)})
+	# Vi skal derivere $f(x) = x^4 - x +2$. Vi bruker regelen $f(x) = x^n \Rightarrow f'(x) = nx^{n-1}$. Vi får da at $f'(x) = 4x^3 - 1$.
+	
+	# Her ser vi at funksjonen g er sammensatt av to funksjoner som er multiplisert sammen, nemlig $x^3$ og $\ln(x)$. Vi bruker derfor produktregelen: $f(x) = uv \Rightarrow f'(x) = u'v + uv'$. Vi får da 
+	\begin{equation*}
+		\begin{aligned}
+			g'(x) &= 3x^2 \cdot \ln(x) + x^3 \cdot \frac{1}{x}\\
+					&= 3x^2\ln(x) + x^2\\
+					&= x^2(3\ln(x)+1)\\
+		\end{aligned}
+	\end{equation*}
+	
+	# Her får vi bruk for kjerneregelen, der vi velger at kjernen vår er $u = 2x^2 + x$. Vi har at 
+	\begin{equation*}
+		\begin{aligned}
+				h(x) = e^{u(x)} \Rightarrow h'(x) &= (e^{u(x)})' \cdot u'(x) \\
+																  &= e^{u(x)} \cdot (4x + 1) \\
+																  &= (4x +1)e^{2x^2 + x}
+		\end{aligned}
+	\end{equation*}
+\end{easylist}
+
+
+\subsection*{Oppgave 2}
+\begin{easylist}[enumerate]
+	\ListProperties(Style2*=,Numbers=a,Numbers1=l,FinalMark={)})
+	# 
+	\begin{equation*}
+		\frac{1}{2x-2} + \frac{2}{x-3} - \frac{x-2}{x^2 - 4x +3}
+	\end{equation*}
+	Først faktoriserer vi nevnerene for å finne ut hva fellesnevneren til brøkene er. Nevneren i det første leddet faktoriseres slik: $2x-2 = 2(x-1)$. Nevneren i andre legg kan ikke faktoriseres, mens nevneren i det tredje leddet kan vi faktorisere for eksempel ved bruk av abc-formelen. Etter faktoriseringen ser uttrykket ut slik
+	
+	\begin{equation*}
```

Comment:
Tror ikke du trenger å ha align inne i equation. Se: https://en.wikibooks.org/wiki/LaTeX/Advanced_Mathematics#align_and_align*

Reply:
Enig, ser ut som man bruker align istedet for equation hvis man vil ha likninger over flere linjer.